tests/integration/cairo1/misc/test_panic/panic_test.cairo

```diff
@@ -0,0 +1,5 @@
+#[test]
+fn test_panic() {
+    let temp = 1234;
+    assert(1 == temp, temp);
+}
```

Comment:
Newline missing

Reply:
That's funny but it seems that GitHub is not displaying empty lines at the end of the file